src/gui/addnewtorrentdialog.ui

```diff
@@ -195,6 +188,52 @@
             </item>
            </layout>
           </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_2">
+            <item>
+             <widget class="QLabel" name="contentLayoutLabel">
+              <property name="text">
+               <string>Content layout:</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QComboBox" name="contentLayoutComboBox">
+              <property name="currentIndex">
+               <number>0</number>
+              </property>
+              <item>
+               <property name="text">
+                <string>Default</string>
```

Comment:
imho "Default" is bit confusing here, "Default" of what? may be rename it to "Don't touch anything" or maybe someone else can think of better wording. or maybe place it under enable/disable checkbox ("Change root folder") and remove the "Default" option

Reply:
>"Default" of what?

"Default" of current torrent.
>"Change root folder"

Using any kind of "Change", "Strip" is bad, IMO. Really we can't say we want to change it in any cases when we enable it in application preferences.
https://github.com/qbittorrent/qBittorrent/pull/13955#issuecomment-748913288

Reply:
> "Default" of current torrent.

yes, I was able to infer it but IMO it will be a bit difficult for a non-technical user or a nonnative speaker.

> Using any kind of "Change", "Strip" is bad, IMO. Really we can't say we want to change it in any cases when we enable it in application preferences.
> [#13955 (comment)](https://github.com/qbittorrent/qBittorrent/pull/13955#issuecomment-748913288)

reading [#13955 (comment)](https://github.com/qbittorrent/qBittorrent/pull/13955#issuecomment-748913288), I agree with you but still using "Default" doesn't seem right to me. generally "Default" infers a configurable option in the main settings and when "Default" is selected we use the value from there.